### PR TITLE
coordinate multiple flows

### PR DESCRIPTION
**Pull request by tomisacat**
Add static functions to resolve issue #8

### CODE REVIEW

File: RxFlow/RxFlow/Flow.swift [hunk 1]
```diff
@@ -64,7 +64,26 @@ extension Flow {
...
 public class Flows {
-
+    /// Allow to be triggered only when Flows given as parameters are ready to be displayed.
+    /// Once it is the case, the block is executed
+    ///
+    /// - Parameters:
+    ///   - flows: Flow(s) to be observed
+    ///   - block: block to execute whenever the Flows are ready to use
+    public static func whenReady<RootType: UIViewController>(flows: [Flow],
+                                                             block: @escaping (_ warpRoots: [RootType]) -> Void) {
```
Comment (twittemb) [inline RxFlow/RxFlow/Flow.swift]: can you rename warpRoots in flowRoots please ?

Comment (tomisacat) [reply RxFlow/RxFlow/Flow.swift]: Sure : )

File: RxFlow/RxFlow/Flow.swift [hunk 2]
```diff
@@ -64,7 +64,26 @@ extension Flow {
...
+        let flowObservables = flows.map { $0.rxFlowReady.asObservable() }
+        _ = Observable<Void>.zip(flowObservables, { (_) in
+            return Void()
+        }).take(1).subscribe(onNext: { (_) in
+            let optionalRoots = flows.map { $0.root as? RootType }.filter { $0 != nil }
+            guard let roots = optionalRoots as? [RootType],
```
Comment (m-de-chalendar-betclic) [inline RxFlow/RxFlow/Flow.swift]: You could simplify this by using `let roots = flows.flatMap { $0.root as? RootType }`. `roots` will be inferred as `[RootType]`.  

I have nothing to do with RxFlow though, was just checking the project, so you might want to wait for @twittemb's opinion on this.

Comment (tomisacat) [reply RxFlow/RxFlow/Flow.swift]: @m-de-chalendar-betclic You're right : )

**Comment by twittemb** [comment]
Thanks @m-de-chalendar-betclic for your excellent remark. I will review this today and merge this just after if the tests are OK. Thanks @tomisacat for this PR.

**Comment by tomisacat** [comment]
@twittemb  OK. working on it

**Comment by tomisacat** [comment]
@twittemb Done. BTW, I'm not familiar with `git rebase`, if there's something wrong please let me know : )

**Comment by twittemb** [comment]
Hi @tomisacat 

It seems good to me. The CI has to succeed before I can merge the PR. It will be merged within the next hours.

Thanks again for you contribution.

**Comment by twittemb** [comment]
@tomisacat thanks for this PR.

**Pull request merged**